INSTALL.md

```diff
@@ -0,0 +1,67 @@
+# How to install MacGitver
+
+In present state, you need to compile MacGitver from source. We provide a compilable version from our default branch `development`. If you like to help with creating binary packages, please create a [pull request](https://github.com/macgitver/MacGitver/pulls) or contact me via [mail](mailto:nils@macgitver.org).
```

Comment:
Please create an issue in Redmine to push me hard to get that mailing list working xD